build.gradle.kts

```diff
@@ -184,14 +184,19 @@ subprojects {
         val isLocalizationsProject = project.name == "localization"
         lintConfig = rootProject.file(if (isLocalizationsProject) "lint-no-op.xml" else "lint.xml")
```

Comment:
This part requires further consideration. During the release process, Lint will block us if there are any missing translations or other issues, which would prevent us from building release builds. I don't think it’s feasible to expect release managers to handle this during release process.

We should explore a way to exclude Lint issues related to the `:localization` project if we decide to aggregate the results.

Reply:
Thank you for additional context, I've missed this.

Would something like this: eafe896 work for you? I think disabling lint in a project is project-specific configuration, so I put it directly there.

Reply:
Yup, this works. Thanks!